packages/now-cli/src/util/dev/server.ts

```diff
@@ -1398,13 +1385,23 @@ export default class DevServer {
     });
 
     if (statusCode) {
-      res.statusCode = statusCode;
+      // Set the `statusCode` as read-only so that `http-proxy`
+      // in not able to modify the value in the future
```

Comment:
```suggestion
      // is not able to modify the value in the future
```

I think there is a typo here. Also, why would `http-proxy` assign the status code?